pom.xml

```diff
@@ -5,28 +5,79 @@
 
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+      <groupId>io.airlift</groupId>
+      <artifactId>airbase</artifactId>
+      <version>144</version>
+    </parent>
+
     <groupId>io.trino.gateway</groupId>
     <artifactId>trinogateway-parent</artifactId>
-    <name>trinogateway-parent</name>
     <packaging>pom</packaging>
     <version>3-SNAPSHOT</version>
 
+    <name>trinogateway-parent</name>
+    <description>A smart load balancer and query route for Trino</description>
+    <url>https://github.com/trinodb/trino-gateway</url>
+    <inceptionYear>2018</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <developers>
+      <developer>
+        <name>Trino Gateway contributors</name>
+        <url>https://github.com/trinodb/trino-gateway/graphs/contributors</url>
```

Comment:
4 spaces for this file also

Reply:
done .. so sorry .. four is just sooo wide